src/tinypeel/tinypeel.py

```diff
@@ -17,7 +17,14 @@ def runPeelingCycles(pedigree, peelingInfo, args, singleLocusMode=False):
     # Right now maf _only_ uses the penetrance so can be estimated once.
     if args.est_alt_allele_prob:
         PeelingUpdates.updateMaf(pedigree, peelingInfo)
-
+    if args.alt_allele_prob_file is not None:
+        pedigree.readInAAP(pedigree, args.alt_allele_prob_file)
```

Comment:
I think I can remove this line (line 21) as it is already triggered in tinyhouse/InputOutPut Line 237 to 239 (https://github.com/AlphaGenes/tinyhouse/pull/15/commits/a276aab24976c12d24524a192ab8aaf109bfb147). Is this correct @XingerTang?

Reply:
@RosCraddock You are right.